src/content/documentation/develop/web-ext-command-reference.md

```diff
@@ -474,10 +497,24 @@ Environment variable: `$WEB_EXT_TIMEOUT`
 
 #### `--id`
 
-A custom identifier string for the extension. This has no effect if the extension already declares an identifier in its manifest. This option may be useful for signing versions of an exisiting extension that you own.
+A custom identifier string for the extension. This has no effect if the extension already declares an identifier in its manifest. This option may be useful for signing versions of an existing extension that you own. 
+
+::: note alert
+This variable cannot be used with the add-on submission API - custom identifers must be specified in the manifest.
```

Comment:
```suggestion
This option cannot be when `--use-submission-api` is also used: the add-on ID must be specified in the `manifest.json` file.
```

Reply:
generally agree with the suggestion, but it's got a grammar mistake (missing `used`?)

Reply:
ugh, sorry. fixed